docs/concepts/key-terms/tracing/distributed-tracing.mdx

```diff
@@ -21,7 +21,45 @@ It's important to note that tracing is not [profiling](https://github.com/produc
 
 Learn more in this [Tracing: Frontend issues with backend solutions](https://sentry.io/resources/tracing-frontend-issues-with-backend-solutions/?original_referrer=https%3A%2F%2Fblog.sentry.io%2F) workshop.
 
-### Why Tracing?
+### The Tracing Data Model
+
+Ultimately, any data structure is defined by the kind of data it contains, and relationships between data structures are defined by how links between them are recorded. Traces, transactions, and spans are no different.
+
+#### Traces
+
+Traces are defined as the collection of all transactions that share a `trace_id` value.
+
+#### Transactions
+
+Transactions share most properties (start and end time, tags, etc.) with their root spans. They also have a `transaction_name` property, used in the UI to identify the transaction. Common examples include endpoint paths for backend request transactions, task names for cron job transactions, and URLs for page-load transactions.
+
+Before the transaction is sent, the `tags` and `data` properties will get merged with data from the global scope. (Global scope data is set in `Sentry.init()` or by using `Sentry.configureScope()`, `Sentry.setTag()`, `Sentry.setUser()`, and `Sentry.setExtra()`.)
```

Comment:
Should this be an 'or'? It reads a bit like I would need to use all those functions combined to set the data.

Reply:
Yes!